packages/interaction-kit/src/cli/deploy.ts

```diff
@@ -0,0 +1,28 @@
+import * as API from "../api";
+import { getApplicationEntrypoint } from "../scripts";
+
+export default async function command(argv?: string[]) {
+	if (argv?.includes("--help")) {
+		console.log(`
+			Description
+				Creates all new commands, updates all changed commands, and deletes all removed commands from Discord.
```

Comment:
"Registers" fits better in my opinion.
```suggestion
				Registers all new commands, updates all changed commands, and deletes all removed commands from Discord.
```